napari/components/_viewer/view/main.py

```diff
@@ -26,9 +28,10 @@ def __init__(self, viewer):
         )
 
         self.viewer = viewer
-
-        self.canvas = SceneCanvas(keys=None, vsync=True)
-        self.canvas.native.setMinimumSize(QSize(100, 100))
+        # set vispy application to the qtpy backend
+        app = Application('qtpy')
```

Comment:
seems kinda weird to just initialize this here and never use it again. would prefer `use_app` instead

Reply:
switched to `use_app`